### PR TITLE
#3619 - make worker/runner protocol public

### LINKED ISSUE #3619: workerproto is internal and can't be consumed by other Go projects
**Describe the bug**
`internal/workerproto` contains an implementation of the protocol used by worker-runner to talk to workers. Due to being in an `internal` directory, it can't be used in out-of-tree projects.

**Expected behavior**
`workerproto` should be available for out-of-tree worker implementations.

**Taskcluster version**
v37.3.0

### PR DESCRIPTION
**Pull request by djmitche**
/cc @ricky26  :)

Github Bug/Issue: Fixes #3619

### CODE REVIEW

File: changelog/issue-3619.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+audience: developers
+level: patch
+reference: issue 3619
+---
+The `tools/workerproto` Go package is now available for external use, and its API is considered stable (in other words, breaking changes will result in a major versio bump).
```
[the remark this answers is not in the document]
Comment (djmitche) [reply changelog/issue-3619.md:5]: I swear I read that over for typos..

**Review by helfi92 — APPROVED** [review]

**Comment by helfi92** [comment]
A couple of test failures, otherwise LGTM.

**Comment by djmitche** [comment]
wow, how did I make a commit with no message!

**Comment by djmitche** [comment]
Ok, a goimports run was required since the lexical order changed.  Fixed.

**Pull request merged**